src/HttpServiceProvider.php

```diff
@@ -133,9 +136,9 @@ protected function configureSwooleServer()
         $config = $this->app['config']->get('swoole_http.server.options');
 
         // only enable task worker in websocket mode
-        if (! $this->isWebsocket) {
-            unset($config['task_worker_num']);
-        }
+        // if (! $this->isWebsocket) {
+        //     unset($config['task_worker_num']);
+        // }
 
```

Comment:
@albertcht What effect will it have if I comment it?

Reply:
Because originally task workers will be only enabled for websocket's message pushing, now you can add another condition for detecting if the queue driver is swoole's async task.

Reply:
This is just to avoid launching unnecessary task workers if there's no async task jobs.